packages/components/src/tab-panel/index.tsx

```diff
@@ -25,7 +25,7 @@ const TabButton = ( {
 }: TabButtonProps ) => (
 	<Button
 		role="tab"
-		tabIndex={ selected ? null : -1 }
+		tabIndex={ selected ? undefined : -1 }
```

Comment:
Sorry, this expands the scope outside the `components/src/button/` directory.

But `tabIndex` doesn't accept `null`, only `number | undefined`